src/plyr-audio.vue

```diff
@@ -40,7 +46,15 @@
     },
     mounted () {
       this.player = plyr.setup(document.getElementById(`js-player-audio-${this.idNumber}`), this.options)[0]
+      this.emit.forEach(element => {
+        this.player.on(element,this.emitPlayerEvent)
```

Comment:
Is the emitPlayerEvent function necessary?

Edit: Oh I see it's so you can emit the proper event type and then the proper payload I get it.